rlp/Cargo.toml

```diff
@@ -8,6 +8,7 @@ authors = ["Parity Technologies <admin@parity.io>"]
 edition = "2018"
 
 [dependencies]
+bytes = "0.6"
```

Comment:
we should put `bytes/std` behind `std` feature

Reply:
fixed